src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js

```diff
@@ -0,0 +1,246 @@
+                        >
+                            <Media.Menu
+                                isDisabled={isConfirmingDelete}
+                                data-testid="comment-actions-menu"
+                                menuProps={{
+                                    'data-resin-component': ACTIVITY_TARGETS.COMMENT_OPTIONS,
+                                }}
+                            >
+                                {canEdit && (
+                                    <MenuItem
+                                        data-resin-target={ACTIVITY_TARGETS.COMMENT_OPTIONS_EDIT}
+                                        data-testid="edit-comment"
+                                        onClick={handleEditClick}
+                                    >
+                                        <IconPencil color={bdlGray} />
+                                        <FormattedMessage {...messages.annotationActivityEditMenuItem} />
+                                    </MenuItem>
+                                )}
+                                {canDelete && (
+                                    <MenuItem
+                                        data-resin-target={ACTIVITY_TARGETS.COMMENT_OPTIONS_DELETE}
+                                        data-testid="delete-comment"
+                                        onClick={handleDeleteClick}
+                                    >
+                                        <IconTrash color={bdlGray} />
+                                        <FormattedMessage {...messages.annotationActivityDeleteMenuItem} />
+                                    </MenuItem>
+                                )}
+                            </Media.Menu>
+                            {isConfirmingDelete && (
+                                <DeleteConfirmation
+                                    data-resin-component={ACTIVITY_TARGETS.COMMENT_OPTIONS}
+                                    isOpen={isConfirmingDelete}
+                                    message={messages.annotationActivityDeletePrompt}
+                                    onDeleteCancel={handleDeleteCancel}
+                                    onDeleteConfirm={handleDeleteConfirm}
+                                />
+                            )}
+                        </TetherComponent>
+                    )}
+                    <div className="bcs-AnnotationActivity-headline">
+                        <UserLink
+                            data-resin-target={ACTIVITY_TARGETS.PROFILE}
+                            id={createdByUser.id}
+                            name={createdByUser.name}
+                            getUserProfileUrl={getUserProfileUrl}
+                        />
+                    </div>
+                    <div>
+                        <ActivityTimestamp date={createdAtTimestamp} />
+                    </div>
+                    {isEditing ? (
+                        <CommentForm
```

Comment:
Do we want to reuse this component? Or should we create a new one with just the pieces we need?

Reply:
I think this one handles the cases we need and then some so I re-used it.